CHANGELOG.md

```diff
@@ -11,6 +11,114 @@ git log --first-parent
 
 Please visit our wiki [Changelog](https://github.com/ginkgo-project/ginkgo/wiki/Changelog) for unreleased changes.
 
+## Version 1.9.0
+
+The Ginkgo team is proud to announce the new Ginkgo minor release 1.9.0.
+This release brings new features such as:
+- Supporting half precision (IEEE FP16). The type `gko::half` can now be selected in most instances as the value type
```

Comment:
For the highlights, I would have the same language as for the previous release. Adding an inflection is not necessary. Same for the other points below. (Providing -> Provide, Adding -> Add, Simplifying -> Simplify)
```suggestion
- Support for half precision (IEEE FP16). The type `gko::half` can now be selected in most instances as the value type
```